src/path.cpp

```diff
@@ -206,6 +206,14 @@ namespace {
 		// Fallback to GetFileInformationByHandle for symlinks
 		if (!(flags & dont_follow_links) && (data.dwFileAttributes & FILE_ATTRIBUTE_REPARSE_POINT))
 		{
+#ifdef TORRENT_WINRT
+
+			ec.assign(ERROR_FILE_SYSTEM_LIMITATION, system_category());
```

Comment:
is it better to hard-code this error than to let `CreateFile()` fail? Are you guaranteed that this will fail in WinRT?

Reply:
I've tried to implement symlinks with CreateFile2 and GetFileInformationByHandleEx. It seems to behave well with a symlinked file.